src/Symfony/Component/Notifier/Bridge/Sendberry/README.md

```diff
@@ -0,0 +1,25 @@
+Sendberry Notifier
+===============
+
+Provides [Sendberry](https://sendberry.com/) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+SENDBERRY_DSN=sendberry://NAME:PASSWORD@default?key=KEY&from=FROM
+```
+
+where:
+ - `KEY` is your authentication key generated by the platform.
+ - `NAME` is your user-defined access name.
+ - `PASSWORD` is your user-defined access password.
+ - `FROM` is your sender name (phone number or sender name).
```

Comment:
Please use the following order:
NAME, PASSWORD, KEY, FROM

I would rename NAME to USERNAME